tools/canu/canu.xml

```diff
@@ -56,21 +56,25 @@
         genomeSize='$genomeSize'
         minThreads=\${GALAXY_SLOTS:-4}
         maxThreads=\${GALAXY_SLOTS:-4}
-        redMemory=\${GALAXY_MEMORY_MB:-4096}M
+        redMemory=\${GALAXY_MEMORY_MB:-6144}M
         redThreads=\${GALAXY_SLOTS:-4}
         obtovlThreads=\${GALAXY_SLOTS:-4}
         utgovlThreads=\${GALAXY_SLOTS:-4}
         batThreads=\${GALAXY_SLOTS:-4}
-        batMemory=\${GALAXY_MEMORY_MB:-4096}M
-        cormhapMemory=\${GALAXY_MEMORY_MB:-4096}M
-        obtovlMemory=\${GALAXY_MEMORY_MB:-4096}M
-        utgovlMemory=\${GALAXY_MEMORY_MB:-4096}M
+        batMemory=\${GALAXY_MEMORY_MB:-6144}M
+        cormhapMemory=\${GALAXY_MEMORY_MB:-6144}M
+        obtovlMemory=\${GALAXY_MEMORY_MB:-6144}M
+        utgovlMemory=\${GALAXY_MEMORY_MB:-6144}M
         corThreads=\${GALAXY_SLOTS:-4}
-        corMemory=\${GALAXY_MEMORY_MB:-4096}M
+        corMemory=\${GALAXY_MEMORY_MB:-6144}M
         cnsThreads=\${GALAXY_SLOTS:-4}
-        cnsMemory=\${GALAXY_MEMORY_MB:-4096}M
-        oeaMemory=\${GALAXY_MEMORY_MB:-4096}M
+        cnsMemory=\${GALAXY_MEMORY_MB:-6144}M
+        oeaMemory=\${GALAXY_MEMORY_MB:-6144}M
         oeaThreads=\${GALAXY_SLOTS:-4}
+        merylThreads=\${GALAXY_SLOTS:-4}
+        merylMemory=\${GALAXY_MEMORY_MB:-6144}M
```

Comment:
I would revert it because I think that Meryl requites to provide the memory in Gigabytes.